ophyd/epics_motor.py

```diff
@@ -145,8 +145,18 @@ def move(self, position, wait=True, **kwargs):
             If motion fails other than timing out
         '''
         self._started_moving = False
+        status = None
```

Comment:
Is it needed here? `None` is not valid within the callback below, and overwritten on line 158 anyway.

Reply:
not needed here, deleted it